<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSpaceWin</class>
 <widget class="QWidget" name="NetworkSpaceWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MainBoard" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QListWidget" name="FileListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SpaceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>空间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TransferInfoButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>传输情况</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownloadButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
   <widget class="QListWidget" name="TransferInfoListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DelButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <zorder>SpaceButton</zorder>
   <zorder>TransferInfoButton</zorder>
   <zorder>DownloadButton</zorder>
   <zorder>UploadButton</zorder>
   <zorder>TransferInfoListWidget</zorder>
   <zorder>FileListWidget</zorder>
   <zorder>DelButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
